docs/development_suite/deploy/gitops-based/index.md

```diff
@@ -0,0 +1,35 @@
+---
+id: gitops-based
+title: GitOps deployment strategy
+sidebar_label: GitOps deployment strategy
+---
+
+:::caution
+This is a **BETA** feature and, as such, is currently under active development. Pay attention using it.
+:::
+
+**GitOps** is a modern approach to managing and automating the deployment of applications and infrastructure in a software development lifecycle. 
+By enabling the new **Enhanced Project Workflow** <!-- TODO: LINK -->on a Console project, it is possible to adopt a GitOps deployment strategy instead of the classic pipeline-based one.
+
+## Introduction to GitOps
+
+GitOps deployment is based on the principle of using Git repositories as the single source of truth for both application code and deployment configurations.
+
+Popular tools that support the GitOps deployment strategy include ArgoCD, Flux, Jenkins X, and more. Such tools operate by comparing the desired state of your project, as defined in the project’s Git repository, with the actual state of your Runtime resources. If there is a discrepancy, the tool automatically takes corrective actions to bring the cluster's state in line with the desired state.
```

Comment:
```suggestion
Popular tools that support the GitOps deployment strategy include ArgoCD, Flux, Jenkins X, and more. Such tools operate by comparing the desired state of your project, as defined in the project’s Git repository, with the actual state of your Runtime resources. If there is a discrepancy, the tool may automatically take corrective actions to bring the cluster's state in line with the desired state.
```

taking action depends on the tool configuration; e.g. ArgoCD detects changes from git and applies them but if you manually change something on the cluster it's no guaranteed that argo would update the configuration, it depends on how you have configured the specific configuration.

For this reason I'd use *may automatically*